core/src/dynamic.rs

```diff
@@ -19,16 +16,16 @@ pub use scale_value::{At, Value};
 /// for dynamic requests.
 pub type DecodedValue = scale_value::Value<u32>;
 
-// Submit dynamic transactions.
+/// Submit dynamic transactions.
 pub use crate::tx::dynamic as tx;
 
-// Lookup constants dynamically.
+/// Lookup constants dynamically.
 pub use crate::constants::dynamic as constant;
 
-// Lookup storage values dynamically.
+/// Lookup storage values dynamically.
 pub use crate::storage::dynamic as storage;
 
-// Execute runtime API function call dynamically.
+/// Execute runtime API function call dynamically.
```

Comment:
Nit: IIRC adding comments to types like this that have more comments leads to a concatenation of this comment + the one on the actual type (but I'd have to actually check that!). That may be why they weren't doc comments before